### DISCUSSION TITLE
Adding a stage from file - root node naming convention

### DISCUSSION THREAD

**Discussion by sshrestha-fractal** [Autodesk/maya-usd#1379] [General] [answered]
Hi,

When I add a stage from a usd file into Maya, it seems to auto-capitalize the first letter of the file name. This creates the root hierarchy node with upper camel case where-as we use lower camerl case in our pipeline.

Here is an example for a file called woodenTentA_model:
![image](https://user-images.githubusercontent.com/77970661/116659428-36e0b880-a9af-11eb-87a7-ac82a47399bd.png)

Is there a way to control this behaviour?

Thanks,
Sachin

**Comment by womanyoyoyo-adsk** [comment, marked as answer]
Hi @sshrestha-fractal, 

I'm going to internally log this to lift the capitalization off the mayaUsdProxyShape when Creating Stage from File. Thanks for reporting. 

Natalia
